Week7/MAKEME.md

```diff
@@ -1,122 +1,75 @@
 # Homework Week 7
 
->[Here](https://github.com/HackYourFuture/JavaScript/tree/master/Week7/README.md) you find the readings you have to complete before the eighth lecture.
+```
+Topics discussed this week:
+• Object Oriented Programming
```

Comment:
Do we teach about ES6 Classes in this lecture? If so, we need to add it to the homework. This would also allow practicing `this`.